addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -90,12 +94,13 @@ options.registry.SnippetPopup = options.Class.extend({
             case 'moveBlock':
                 return this.$target.closest('footer').length ? 'moveToFooter' : 'moveToBody';
             case 'setLayout':
-                if (this.$target.hasClass('s_popup_center')) {
-                    return 'modal';
-                } else if (this.$target.hasClass('s_popup_fixed_top')) {
-                    return 'fixedTop';
+                if (this.$target.hasClass('s_popup_middle')) {
+                    return 'Middle';
+                } else if (this.$target.hasClass('s_popup_top')) {
+                    return 'Top';
+                } else if (this.$target.hasClass('s_popup_bottom')) {
+                    return 'Bottom';
                 }
```

Comment:
That whole `case` clause will become useless with what I explained above, data-select-class will do the job automatically